### PR TITLE
chore(CPlusPlus): add minimum swaps to make elements less than K together

### PR DESCRIPTION
**Pull request by Shivang-Konde**
**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix
- [x] New algorithm
- [ ] Optimization in previous algorithms
- [ ] Code style update
- [ ] Refactor
- [ ] Documentation
- [ ] Other, please describe:

**Briefly describe the changes in this PR**
I have added code to find out how many minimum swaps are needed to make all elements which are less than K together . 

**Other information:**

### CODE REVIEW

File: algorithms/CPlusPlus/Arrays/Minimum_swaps_to_make_elements_less_than_k_together.cpp [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+        cin>>n;
+        int arr[n];
+        for(int i=0;i<n;i++)
+            cin>>arr[i];
+        cin>>k;
+        cout << minSwap(arr, n, k) << "\n";
+    }
+	return 0;
+}
+
```
Comment (UG-SEP) [inline algorithms/CPlusPlus/Arrays/Minimum_swaps_to_make_elements_less_than_k_together.cpp:60]: Add the time complexity and a test-case example

File: algorithms/CPlusPlus/Arrays/Minimum_swaps_to_make_elements_less_than_k_together.cpp [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+	int t,n,k;
+	cin>>t;
+	while(t--)
+    {
+        cin>>n;
+        int arr[n];
+        for(int i=0;i<n;i++)
+            cin>>arr[i];
+        cin>>k;
```
Comment (UG-SEP) [inline algorithms/CPlusPlus/Arrays/Minimum_swaps_to_make_elements_less_than_k_together.cpp:55]: print user-friendly info that what to enter 
> i.e. cout<<"Enter the number of test cases ";

**Review by UG-SEP — CHANGES_REQUESTED** [review]
Please kindly add your program name and link in c++ [readme.md](https://github.com/MakeContributions/DSA/blob/main/algorithms/CPlusPlus/README.md)
If you have any doubt regarding this PR free to ask 😊
as well as one more thing that renames the file name from 
Minimum_swaps_to_make_elements_less_than_k_together.cpp to minimum-swaps-and-k-together.cpp

**Comment by ming-tsai** [comment]
Hi @Shivang-Konde,
Any updated?

**Comment by ming-tsai** [comment]
Hi @Shivang-Konde,
This pull request will be closed, any updates please re-open this pull request.

**Pull request closed without merging**